src/threads-ext.js

```diff
@@ -322,6 +335,31 @@ Process.prototype.getCostumeFromRPC = function (url, params) {
     this.pushContext();
 };
 
+Process.prototype.getAudioFromRPC = function(url, params) {
+    var audio;
+
+    //Create Snap XML from input
+    
+        var resultado =  this.callRPC(url, params, true);
+
+        var bufferToBase64 = function (buffer) {
+            var bytes = new Uint8Array(buffer);
+            var len = buffer.byteLength;
+            var binary = "";
+            for (var i = 0; i < len; i++) {
+                binary += String.fromCharCode(bytes[i]);
+            }
+            return window.btoa(binary);
+        };
+        
+        // Clear request
+        audio = this.requestedAudio;
+        this.requestedAudio = null;
+        base64Audio = window.btoa(audio);
+        const result = `<sound collabId=\"item_-1_2\" name=\"audio\"sound=\"${base64Audio}\" id=\"1\"/>`;
```

Comment:
@brollb Should he be using a fixed collab ID here?

Reply:
I don't think so. It could be problematic after multiple are received if you are able to perform an action on them (though that might not be possible).

Reply:
It doesn't look like this is actually called anywhere so I would opt for removing it entirely.